src/models/client.ts

```diff
@@ -205,32 +231,60 @@ export class ClientEnvironment {
     */
   }
 
-  public async clearBrowserCache() {
-    await this.network.clearBrowserCache();
+  private async buildCapturedResponse(response: Network.ResponseReceivedParameters): Promise<CapturedResponse> {
+    return {
+      responseMeta: response.response,
+      body: (await this.network.getResponseBody({requestId: response.requestId})).body
+    };
+  }
+
+  /**
+   * Load page in active tab. Resolves on HTML load. To wait for a full page load, and
```

Comment:
I think we should specify that this method waits for the HTTP response. I think this is more explicit than "HTML load". I also think we should specifically callout that "full page load" means waiting for the `load` event.

Reply:
cleaned up

Reply:
Ha, too many similar acronyms ^_^. All I could read was HTTP. 

I've changed all references of `HTML response` to `initial HTTP response`. I tried just `HTTP response`, but `Resolves on HTTP response` sounded a little vague as to which HTTP response it was referring to.